lib/core/query.js

```diff
@@ -125,16 +125,18 @@ function jsonQuery (id, filters, query, paginate) {
 
   // convert adapterQuery
   if (!_.isEmpty(adapterQuery)) {
-    result.filter = convertOperators(adapterQuery);
+    result.filter = convertOperators(adapterQuery, false, escapeFn);
   }
   debug(result);
   return result;
 }
 
-function convertOperators (query, root = false) {
+function convertOperators (query, root = false, escapeFn) {
```

Comment:
Possible we should move the argument `root` to the end because it is an optional argument.  This will make it unnecessary to provide always the default value when it is not required. Even I would say `root` should only be string or undefined. At the end you can call in the most cases `convertOperators(value,  escapeFn)`.

Reply:
absolutely brainless copy&past